src/app/core/building/building.component.html

```diff
@@ -511,7 +511,7 @@ <h4>3. 指定容積率と比較し、小さい方を採用</h4>
                 <tbody>
                   <tr>
                     <td>容積率の上限</td>
-                    <td class='text-right'>{{result?.buildingSite?.floorAreaRatioLimit}}%</td>
+                    <td class='text-right'>{{result?.buildingSite?.minFloorAreaRatioLimit}}%</td>
```

Comment:
@thanhtetaung  After I add it API response, I'll tell you that.

Reply:
@k-kin Thanks. After you have finished, I will update Java backend and merge this PR.

Reply:
@thanhtetaung  I deployed it to API. ( you mentioned not me but kin )

Reply:
@hirao-k Thanks.